Update PowerToys docs

noraa-junker opened the pull request:
Updated PowerToys docs to match current version. Some settings were missing and I added/updated some images.

noraa-junker committed:
Update index.md

noraa-junker committed:
Update administrator.md

noraa-junker committed:
Update always-on-top.md

noraa-junker committed:
Update PT docs

noraa-junker committed:
Update general.md

noraa-junker committed:
update TOC

Court72 commented:
@mattwojo 

Can you review the proposed changes? Note that the commits will need to be moved into the private repo for staging and automated checks as well. Let us know if you want help with that.

When the changes are ready for publication, add a `#sign-off` comment to signal that the PR is ready for the review team to merge. 

#label:"aq-pr-triaged"
@MicrosoftDocs/public-repo-pr-review-team

mattwojo commented:
@crutkas - Could you take a look through these updates?

@Aaron-Junker - Thanks for these contributions!! These are awesome! One thing that I noticed that will come up as an issue is the new "General settings" doc that you created. I realize this tab isn't represented in docs and you did a good job outlining what's there. I'm not sure that it's necessary to have a doc on as the content there is fairly self-explanatory, but I'll leave that up to @crutkas. If we do decide to keep it, unfortunately, we are required to have a Microsoft full-time employee assigned in the author and ms.author metadata fields... I've advocated for these fields to be relabeled with an "original-author" metadata field added. Currently these author fields need to represent someone internal who is assigned as the on-going "owner" of the content, responsible for keeping it up-to-date and resolving any PRs or Issues related to the content that come in... so that's me for any PowerToys content. I then triage the issue/PR to the right person connected to the utility/content if I don't have enough expertise or background... so if Clint decides to keep the "General settings" page, we will have to change the author to me, but you would still be listed as a contributor (and anyone who dug into commits would see that you created the page).

Jay-o-Way commented on hub/powertoys/fancyzones.md:
```diff
@@ -129,6 +133,9 @@ Canvas layout also has keyboard support for zone editing. Use the <kbd>arrow</kb
 
 ### Quickly changing between custom layouts
 
+> [!WARNING]
```
There is no danger at all, please use "Note"

crutkas replied:
agree but this could be a visual thing

Jay-o-Way commented on hub/powertoys/awake.md:
```diff
@@ -39,15 +39,14 @@ The following Awake states can be selected:
 
 While PowerToys Awake can keep the computer awake indefinitely or temporarily, in its default state the displays connected to the machine will turn off, even though the computer won't go to sleep. If you need the displays to be available, use the **Keep screen on** switch, which will ensure that all monitors remain on.
 
-This feature only works if Awake is running in one of the two `Keep awake` state.
```
This change isn't right. Please undo.

Jay-o-Way commented on hub/powertoys/file-explorer.md:
```diff
@@ -26,14 +29,33 @@ File Explorer add-ons currently include:
 
 ## Enable PowerToys Thumbnail Previews
 
+> [!NOTE]
+> A reboot might be required after enabling the thumbnail previewer for the settings to take effect.
+> [!WARNING]
+> Thumbnails might not appear on paths managed by cloud storage solutions like OneDrive, since these solutions may get their thumbnails from the cloud instead of generating them locally.
+
```
```suggestion
> [!NOTE]
> A reboot might be required after enabling the thumbnail previewer for the settings to take effect.
> Thumbnails might not appear on paths managed by cloud storage solutions like OneDrive, since these solutions may get their thumbnails from the cloud instead of generating them locally.
```

crutkas replied:
with my tweak and @Jay-o-Way  i like it.  i was about to comment to merge the two

Jay-o-Way commented on hub/powertoys/file-explorer.md:
```diff
@@ -26,14 +29,33 @@ File Explorer add-ons currently include:
 
 To enable preview support, set the extension to **On**.
 
+### Settings fot Stereolithography (.stl) files
```
```suggestion
### Settings for Stereolithography (.stl) files
```

Jay-o-Way commented on hub/powertoys/file-explorer.md:
```diff
@@ -26,14 +29,33 @@ File Explorer add-ons currently include:
 
 Preview Pane is an existing feature in the Windows File Explorer which allows you to see a preview of the file's contents in the view's reading pane. PowerToys adds multiple extensions: Markdown, SVG, PDF, and G-code. In addition to those, PowerToys also adds support for source code files (for more than 150 file extensions).
 
+### Settings fot Source code files previewer
```
```suggestion
### Settings for Source code files previewer
```

Jay-o-Way commented on hub/powertoys/file-explorer.md:
```diff
@@ -26,14 +29,33 @@ File Explorer add-ons currently include:
 
+### Settings fot Source code files previewer
+
+You can expand the **Source code files (Monaco)** section to change certain settings.
```
```suggestion
You can expand the **Source code files (Monaco)** section to change the following settings.
```

Jay-o-Way reviewed (changes requested):
Nice work, but a couple of changes please.

Jay-o-Way commented:
> I'm not sure that it's necessary to have a doc on as the content there is fairly self-explanatory

I totally agree

Jay-o-Way commented on hub/powertoys/administrator.md:
```diff
@@ -33,18 +31,25 @@ These are the two scenarios PowerToys will not work in:
 Admin mode permissions may be required in the following scenarios:
 
+- Always On Top
+  - Pin windows that are elevated
```
Is it the _window_ or the _process_ that is elevated?

Aaron-Junker commented:
> > I'm not sure that it's necessary to have a doc on as the content there is fairly self-explanatory
> 
> I totally agree

@Jay-o-Way @mattwojo  I see the point, but a lot of the things in these docs are already explained in the PowerToys settings and are very self-explanatory. So I think we should add it just to have a complete documentation.

crutkas commented on hub/powertoys/awake.md:
```diff
@@ -39,15 +39,14 @@ The following Awake states can be selected:
 While PowerToys Awake can keep the computer awake indefinitely or temporarily, in its default state the displays connected to the machine will turn off, even though the computer won't go to sleep. If you need the displays to be available, use the **Keep screen on** switch, which will ensure that all monitors remain on.
 
-This feature only works if Awake is running in one of the two `Keep awake` state.
+This feature only works if Awake is running in three of the two `Keep awake` state.
```
Think @jay-o-way this suggestion is correct

```suggestion
This feature only works if Awake is running in one of the three `Keep awake` state.
```

crutkas commented on hub/powertoys/file-explorer.md:
```diff
@@ -26,14 +29,33 @@ File Explorer add-ons currently include:
 ## Enable PowerToys Thumbnail Previews
 
+> [!NOTE]
+> A reboot might be required after enabling the thumbnail previewer for the settings to take effect.
```
```suggestion
> A reboot may be required after enabling the thumbnail previewer for the settings to take effect.
```

crutkas commented:
left some comments, i agree with most of Jay's comments  overall, great work and greatly appreciate diving in

mattwojo committed:
Update hub/powertoys/file-explorer.md

Co-authored-by: Jay <65828559+Jay-o-Way@users.noreply.github.com>

mattwojo committed:
Update hub/powertoys/file-explorer.md

Co-authored-by: Jay <65828559+Jay-o-Way@users.noreply.github.com>

mattwojo committed:
Update hub/powertoys/file-explorer.md

Co-authored-by: Jay <65828559+Jay-o-Way@users.noreply.github.com>

mattwojo committed:
Update hub/powertoys/file-explorer.md

Co-authored-by: Jay <65828559+Jay-o-Way@users.noreply.github.com>

mattwojo committed:
Update hub/powertoys/awake.md

Co-authored-by: Clint Rutkas <clint@rutkas.com>

mattwojo committed:
Update hub/powertoys/file-explorer.md

Co-authored-by: Clint Rutkas <clint@rutkas.com>

mattwojo committed:
Merge branch 'docs' into patch-2

mattwojo committed:
Add metadata description, remove author

The author assigned must be a Microsoft FTE (which defaults to me), but I want to credit Aaron Junker as being the actual original author of this page.

mattwojo committed:
Merge branch 'docs' into patch-2

mattwojo committed:
reorder

mattwojo committed:
typo

mattwojo committed:
fix desc tag

Aaron-Junker commented:
Oh thank  you @mattwojo for finishing this PR! :)
